pkg/combustion/kubernetes_test.go

```diff
@@ -821,3 +821,46 @@ func TestConfigureKubernetes_SuccessfulRKE2ServerWithManifests(t *testing.T) {
 	assert.Contains(t, contents, "name: my-nginx")
 	assert.Contains(t, contents, "image: nginx:1.14.2")
 }
+
+func TestKubernetesVIPManifestValidIPV4(t *testing.T) {
```

Comment:
Nit-picking, but I'd align these too names from "IPV4"/"IPV6") to IPv4 as the rest of the code/prints.